browser/ui/views/bookmarks/brave_bookmark_context_menu_unittest.cc

```diff
@@ -10,6 +10,7 @@
 #include <utility>
 #include <vector>
 
+#include "base/metrics/user_metrics.h"
```

Comment:
Unused?

Reply:
Yes! thanks for pointing out. I thought it is needed by `bookmarks::metrics::`.